### PR TITLE
Feature/csi 1319 create z build jobs in jenkins

### PR DESCRIPTION
**Pull request by ArbelNathan**
CSI-1319

### CODE REVIEW

File: scripts/run_yamlcheck.sh [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #!/bin/bash -xe
+
+arch=$(uname -m)
+echo arch
```
Comment (oriyarde) [inline scripts/run_yamlcheck.sh:4]: remove the echo line

File: scripts/run_yamlcheck.sh [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+if [ "$arch" = "x86_64" ];
+then
 docker run -t -v `pwd`/deploy/kubernetes/v1.13:/deploy/kubernetes/v1.13 garethr/kubeval deploy/kubernetes/v1.13/*.yaml
 docker run -t -v `pwd`/deploy/kubernetes/v1.14:/deploy/kubernetes/v1.14 garethr/kubeval deploy/kubernetes/v1.14/*.yaml
-
+fi
```
Comment (oriyarde) [inline scripts/run_yamlcheck.sh:9]: add `else` echo "skipping k8s yaml validations since it only supports x86" (something like this)

Comment (ArbelNathan) [reply scripts/run_yamlcheck.sh:10]: done

File: scripts/run_yamlcheck.sh (hunk 1 above)
Comment (ArbelNathan) [reply scripts/run_yamlcheck.sh]: done

File: scripts/run_yamlcheck.sh [hunk 3]
```diff
@@ -1,4 +1,10 @@
 #!/bin/bash -xe
+
+arch=$(uname -m)
+if [ "$arch" = "x86_64" ];
```
Comment (igorgonibm) [inline scripts/run_yamlcheck.sh:4]: shoulodn't we skip also for x86/x86_32?

Comment (ArbelNathan) [reply scripts/run_yamlcheck.sh:4]: do we have slaves that work on x86/x86_32?

Comment (ArbelNathan) [reply scripts/run_yamlcheck.sh:4]: do we have any slaves that work on x86/x86_32?

Comment (ArbelNathan) [reply scripts/run_yamlcheck.sh]: change it to be on the safe side

Comment (oriyarde) [reply scripts/run_yamlcheck.sh]: I think they only support 64:
https://hub.docker.com/r/garethr/kubeval/tags

**Pull request merged**